### DISCUSSION TITLE
How to login on Linux?

### DISCUSSION THREAD

**Discussion by ySergi** [crunchy-labs/crunchy-cli#429] [Q&A]
It may sound stupid, but I can't login as I usually do with Windows. I just put `crunchy-cli --credentials "email:password" login` and logs in sucesfully on Windows, but right now I'm on Linux and I can't login... it gives me the following error:
```
:: ✔ Logging in                                                                                               
:: invalid_grant - auth.obtain_access_token.invalid_credentials (https://www.crunchyroll.com/auth/v1/token)
```
Also, anonymous mode doesn't work either.
```
:: ✔ Logged in                                                                                                
:: An error occurred: Anonymous login cannot be saved
```

**Comment by Frooastside** [comment]
i had the same problem and i did something like this, where password_file is a file with only the password in it in your current working directory
```bash
crunchy-cli login --credentials "EMAIL:$(cat password_file)"
```
